docs/content/tutorial/custom-steps.md

```diff
@@ -0,0 +1,154 @@
+# Custom steps for JIRA
+
+In this tutorial, you'll learn how to configure custom steps for use with JIRA. Here's what we'll do with this sample app:
+
+1. Create your app from an app manifest and clone a starter template
+2. Set up and run your local project
+3. Create a workflow using Workflow Builder with custom steps
+4. Create an issue in JIRA using your custom step
+
+## Prerequisites {#prereqs}
+
+Before getting started, you will need the following:
+
+* a development workspace where you have permissions to install apps. If you don’t have a workspace, go ahead and set that up now&mdash;you can [go here](https://slack.com/get-started#create) to create one, or you can join the [Developer Program](https://api.slack.com/developer-program) and provision a sandbox with access to all Slack features for free.
+* a development environment with [Python 3.6](https://www.python.org/downloads/) or later.
+
+**Skip to the code**
+If you'd rather skip the tutorial and just head straight to the code, you can use our [Bolt for Python JIRA functions sample](https://github.com/slack-samples/bolt-python-jira-functions) as a template.
+
+## Creating your app {#create-app}
+
+1. Navigate to the [app creation page](https://api.slack.com/apps/new) and select **From a manifest**.
+2. Select the workspace you want to install the application in, then click **Next**.
+3. Copy the contents of the [`manifest.json`](https://github.com/slack-samples/bolt-python-ai-chatbot/blob/main/manifest.json) file into the text box that says **Paste your manifest code here** (within the **JSON** tab) and click **Next**.
```

Comment:
i'd insert the file directly into the tutorial so they dont have to jump to another page. i can't leave this as a suggestion because it uses backticks oof. pretend those ''' are codeblocks

```
'''js reference title="manifest.json"
https://github.com/slack-samples/bolt-python-ai-chatbot/blob/main/manifest.json
'''
```

that'll make it pull in the manifest.json and keep it up to date